.github/workflows/physionet-build-test.yml

```diff
@@ -26,16 +44,17 @@ jobs:
           apt-get update --yes
       - name: install and configure needed software
```

Comment:
It's kind of nice to have  `ln -sT .env.example .env` step that takes the `.env.example` file and uses this for the test build. Is there a reason why we don't move all/most of the environment variables above to the .env.example?

I could see it may make sense to include the PhysioNet specific variables in this`xxx-build-test.yml` workflow, with the idea that there would be additional workflows for other implementations (e.g. `tcairem-build-test.yml`). If this is the idea, then should we have a separate test workflow that builds from the .env.example defaults?

Reply:
That's a good point, I'll try to extract the common configuration from here and the `docker-compose` file to `.env.example`. There will be slight differences between CI and running locally.